forge-gui-mobile/src/forge/adventure/player/AdventurePlayer.java

```diff
@@ -300,10 +300,15 @@ public void setColorIdentity(ColorSet set) {
 
     @Override
     public void load(SaveFileData data) {
-        clear(); //Reset player data.
+        clear(); // Reset player data.
         this.statistic.load(data.readSubData("statistic"));
         this.difficultyData.startingLife = data.readInt("startingLife");
-        this.difficultyData.startingMoney = data.readInt("startingMoney");
+        // Support for old typo
+        if (data.containsKey("staringMoney")) {
```

Comment:
does the old save work without this check? or just use a starting money from the config file? have you seen the updateDifficulty method used by SaveLoadScene.java for NewGamePlus? if it works without migration then I think it's good to remove this I guess.

Reply:
Yes the old save work without using this. updateDifficulty does assign the value to the variable but it isn't used anywhere. NewGame+ doesn't grant new money.